.goreleaser.yml

```diff
@@ -59,7 +57,6 @@ signs:
       sign
       --dearmor
       --file ${artifact}
-      --signer {{ .Env.SIGNER }}
```

Comment:
One thing i noticed locally is that i get the following error if a `signer` value isn't passed in (even though i have `SIGNER` exported as well as the client id and secret key:
```
⨯ release failed after 15.24s error=sign: sh failed: exit status 1: time="2021-12-14T16:53:55-05:00" level=info msg="Using config file:/Users/apinilla/.signore/config.yaml"
time="2021-12-14T16:53:55-05:00" level=error msg="Error: signer must be set (signore version: v0.1.4-2-g142b976)\nUse `--help` for command information.\n"
```

but should we expected the upstream signore configuration to work as expected?

Reply:
The signer configuration can be read automatically via the `SIGNORE_SIGNER` environment variable (or `signer` field in the signore configuration file, however that option was removed from setup-signore). The reusable workflow sets the environment variable automatically via:

https://github.com/bflad/ghaction-terraform-provider-release/blob/b39aeb3b50c98e4162b57ddcce1337343f6a481f/.github/workflows/hashicorp.yml#L21-L25

https://github.com/bflad/ghaction-terraform-provider-release/blob/b39aeb3b50c98e4162b57ddcce1337343f6a481f/.github/workflows/hashicorp.yml#L110

Reply:
oh cool thanks, I wasn't aware signer could use a `SIGNORE_SIGNER` var instead. in that case, all good here 👍